cloudevents/formats/json-format.md

```diff
@@ -144,6 +144,10 @@ not provided, no assumptions can be made as to the format of the data and
 therefore the `datacontenttype` attribute MUST NOT be present in the resulting
 CloudEvent.
 
+Note: Definition of `data_base64` is a JSON-specific formatting rule and not
```

Comment:
```suggestion
Note: Definition of `data_base64` is a JSON-specific marshaling rule and not
```
@duglin not sure if this is much better, but I think it is slightly more accurate.

Reply:
done!